django/db/models/indexes.py

```diff
@@ -47,23 +44,23 @@ def create_sql(self, schema_editor, suffix=''):
 
         quote_name = schema_editor.quote_name
         return schema_editor.sql_create_index % {
-            "table": quote_name(self.model._meta.db_table),
-            "name": quote_name(self.name),
-            "columns": ", ".join(quote_name(column) for column in columns),
-            "extra": tablespace_sql,
+            'table': quote_name(self.model._meta.db_table),
```

Comment:
You should usually prefer single quotes to double quotes except for strings that use double quotes.